README.md

```diff
@@ -4,16 +4,14 @@
 
 ## Supported JWS Algs
 
-If an `alg` is not specified, this library will assume the following:
+The expected alg will be determined by the following table.
 
 | kty | crvOrSize | alg    |
 | --- | --------- | ------ |
 | OKP | Ed25519   | EdDSA  |
 | EC  | secp256k1 | ES256K |
 | EC  | P-256     | ES256  |
-| RSA | \*        | PS256  |
-
-Other JWS are supported when `alg` is provided by the consuming library.
+| RSA | 4096      | PS256  |
```

Comment:
Nope, overkill... 2048 for a 2020 suite... you won't need the next step up until around 2030 at which point we shouldn't be using RSA anyway.

Reply:
Agreed.